.github/scripts/format_dbt_test_failures.py

```diff
@@ -151,19 +160,116 @@ def rows(self) -> typing.List[typing.List]:
         fieldnames = self.fieldnames
         return [
             [row.get(fieldname) for fieldname in fieldnames]
-            for row in self._rows
+            for row in self.raw_rows
         ]
 
+    def _filter_for_existing_fieldnames(
+        self, possible_fieldnames: typing.List[str]
+    ) -> typing.List[str]:
+        """Helper function to filter a list of `possible_fieldnames` for
+        only those fields that exist in the test group, returning the
+        names of the fields (e.g. ["foo", "bar"])."""
+        existing_fieldnames = self.fieldnames
+        return [
+            field
+            for field in possible_fieldnames
+            if field in existing_fieldnames
+        ]
+
+    def __filter_for_existing_field_indexes(
```

Comment:
**question:** Is the dunderscore use here intentional?

Reply:
Nope, good catch! Fixed in 3642e5d05f7b5b6842c2354f1459b19b0d53b1ec.